covid_moonshot/analysis/fah_plotting.py

```diff
@@ -1,154 +1,191 @@
+import matplotlib.figure
```

Comment:
Not necessary to do this now, but in general, we recommend lazy imports inside methods that use external packages so that if we reuse components of this in other tools, (1) it doesn't take forever to import, and (2) we can have optional dependencies for certain functionality. This also helps enormously when using readthedocs, since we don't need to have all dependencies installed just for sphinx to inspect the docstrings and generate API docs.

Reply:
Makes sense. I'd been doing this mostly with modules that aren't in PyPI / in `install_requires`, but I can be more aggressive about this.